app/helpers/phrase_usage_helper.rb

```diff
@@ -2,4 +2,30 @@ module PhraseUsageHelper
   def highlighted_survey_answer_html(survey_answer, phrase)
     survey_answer.gsub(/(#{phrase})/i){ |match| "<span class='phrase-highlight'>#{match}</span>" }.html_safe
   end
+
+  def phrase_usage_navigation_links
+    links = {}
+
+    if @presenter.pagination.page > 1
+      links = links.merge(
+        previous_page: {
+          url: path_to_previous_page(@presenter.survey_answers_containing_phrase),
+          title: "Previous page",
+          label: "#{@presenter.pagination.page - 1} of #{@presenter.pagination.total_pages}"
+        }
+      )
+    end
+
+    if @presenter.pagination.page < @presenter.pagination.total_pages
```

Comment:
You're violating the law of demeter here, the helper knows too much about the presenter (and the presenter hardly does anything (assuming I'm right in thinking that you're expecting an instance of PhraseUsagePresenter). I'd refactor so the presenter has methods like `page`,`total_pages` etc and the helper only calls `page` or `total_pages` rather than `presenter.pagination.page`

Reply:
This is the same pattern as we currently use elsewhere in the app (i.e. the SummaryPresenter). The presenter _does_ have these methods as you describe, but they're on the PaginationPresenter which is included in the SummaryPresenter.

I'm also following the same pattern here as Content Data: 

https://github.com/alphagov/content-data-admin/blob/master/app/presenters/content_items_presenter.rb

https://github.com/alphagov/content-data-admin/blob/master/app/presenters/pagination_presenter.rb 

The only difference is their ContentItemsPresenter `delegates` to the other presenters.

Reply:
> The presenter does have these methods as you describe, but they're on the PaginationPresenter which is included in the SummaryPresenter.

The helper should neither know nor care how the summary presenter "knows" which page they're on, so calling `.pagination.foo` means that the helper knows this. I'd suggest you SummaryPresenter should delegate these things directly to the PaginationPresenter, then the helper knows less (plus the code is slightly shorter 🎉 )